dask_expr/_collection.py

```diff
@@ -878,12 +878,23 @@ def explode(self, column):
         column = _convert_to_list(column)
         return new_collection(expr.ExplodeFrame(self.expr, column=column))
 
-    def drop(self, labels=None, columns=None, errors="raise"):
-        if columns is None:
-            columns = labels
-        if columns is None:
+    def drop(self, labels=None, axis=0, columns=None, errors="raise"):
+        if columns is None and labels is None:
             raise TypeError("must either specify 'columns' or 'labels'")
-        return new_collection(expr.Drop(self.expr, columns=columns, errors=errors))
+
+        if isinstance(axis, str):
```

Comment:
We have a utility for this

Reply:
Can you point me to it? I looked in `_util.py` w/o anything standing out.

Reply:
Hm yikes it's a method in dask/dask that's defined on the DataFrame class

Please create a utility then, we will need this more often